TradePlay/src/App.js

```diff
@@ -1,33 +1,35 @@
-import { Homepage, Footer, Navbar } from "./components/components";
+import {
+  Homepage,
+  Footer,
+  Navbar,
+  VideoListing,
+} from "./components/components";
```

Comment:
You can use the `index.js` file instead of `components` to export all the files from the component folder. 

Then, you only need to do 

```
import { ... } form './components`
```

It will automatically pick up `index` file

Reply:
Thank you for the suggestion, I will do the changes